R/prototype.R

```diff
@@ -129,3 +129,7 @@ as_vec_ptype.data.frame <- function(x) {
   vec_restore(cols, x)
 }
 
+#' @export
+as_vec_ptype.POSIXlt <- function(x) {
+  as_vec_ptype(as.POSIXct(x))
```

Comment:
This should be implemented using `new_date_time()` and explicitly keep the timezone.